dashboards/lodestar_block_processor.json

```diff
@@ -2187,10 +2194,6 @@
     },
     {
       "collapsed": false,
-      "datasource": {
```

Comment:
why are these removed? seems like something went wrong, how did you export the dashboard?